worlds/stardew_valley/docs/setup_en.md

```diff
@@ -16,6 +16,13 @@
     * It is **not** recommended to further mod Stardew Valley with unsupported mods, although it is possible to do so. Mod interactions can be unpredictable, and no support will be offered for related bugs.
     * The more unsupported mods you have, and the bigger they are, the more likely things are to break.
 
+## Notes About Game Versions
+The randomizer is built for a specific version of Stardew Valley. When a new version gets released, we will try our best to support it as fast as possible, but can't make promises.

```

Comment:
I personally like when pronouns are repeated (but we can't make promises), but I'm not an expert, maybe that's bad writing or sth

Reply:
ooh yeah I uh missed that... that does look better if you do repeat the pronouns (Kaito is Quebecois)

Reply:
Sure, that sounds better imo